### PR TITLE
feat: upgrading stream-chat to v8.2.1

### PR DESCRIPTION
**Pull request by vanGalilea**
## 🎯 Goal

Upgrading `stream-chat` to the latest version prior to release

## 🎨 UI Changes

<details>
<summary>iOS</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

<details>
<summary>Android</summary>

<table>
    <thead>
        <tr>
            <td>Before</td>
            <td>After</td>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                
            </td>
            <td>
                
            </td>
        </tr>
    </tbody>
</table>
</details>

## ☑️ Checklist

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [ ] PR targets the `develop` branch
- [ ] Documentation is updated
- [ ] New code is tested in main example apps, including all possible scenarios
  - [ ] SampleApp iOS and Android
  - [ ] Expo iOS and Android

### CODE REVIEW

**Comment by khushal87** [comment]
Where are the changes of `yarn.lock` of all packages and sample apps due to this? We should push it too IMO. @vanGalilea

**Comment by khushal87** [comment]
PR checks fail because of this I think

**Review by khushal87 — APPROVED** [review]
Approving it, merge if the CI is green :)

**Comment by khushal87** [comment]
@vanGalilea why are there changes in translations files?

File: .github/workflows/check-pr.yml [hunk 1]
```diff
@@ -11,19 +11,30 @@ on:
...
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
       - name: Installation & Build Package
         # installation already takes care of calling prepare which calls build
         # for the sdk package
-        run: yarn --frozen-lockfile && yarn bootstrap-ci
+        run: |
```
Comment (vanGalilea) [inline .github/workflows/check-pr.yml:29]: Manually bootstrapping

**Pull request merged**